wiki/Ranking_criteria/Metadata/ru.md

```diff
@@ -0,0 +1,277 @@
+---
+no_native_review: true
+---
+
+# Метаданные
+
+## Основное
+
+### Правила
+
+- **[Первоисточники метаданных](/wiki/Beatmap/Primary_metadata_source) должны быть использованы в качестве референса.** Вы можете изменять метаданные только в случаях, разрешённых или обязательных для критерий ранкинга. Если первоисточников нет, используйте то, с чем песня больше всего ассоциируется.
+- **Все сложности в мапсете должны иметь одинаковые поля `Title`, `Artist`, `Tag`, `Source` и `BeatmapSetID`.**
+- **Полы `Artist` или `Title` длиннее 81 символа должны быть укорочены.** <!-- this rule matches a technical limitation of BSS and can be removed if the issue is fixed -->
```

Comment:
No need to keep comments on translated articles. Those are left for future changes on the original articles. Translations, however, are fully dependent of said original articles so the annotations aren't needed.
```suggestion
- **Полы `Artist` или `Title` длиннее 81 символа должны быть укорочены.**
```